### ISSUE TITLE
PHP 8.1 Deprecation Warning in Stream

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-security#255]
```
Deprecated | Constant WP_Stream\FILTER_SANITIZE_STRING is deprecated | 1 |  
-- | -- | -- | --

vendor/xwp/stream/classes/class-filter-input.php:100
```

Acceptance criteria:

- [ ] Update the existing fork https://github.com/humanmade/stream
- [ ] The deprecated warnings are fixed in the fork
- [ ] References in composer.json are updated to use the fork

**Comment by mikelittle** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Vulcan-6284e0dfb386b00c83981426/issues/humanmade/altis-security/255?planning-poker) with Zenhub @ferschubert-hm @jerico @kovshenin @wisyhambolu

**Comment by kovshenin** [comment]
Issue has been fixed and committed upstream: https://github.com/xwp/stream/issues/1343

But it has not yet been released. Let's try and avoid creating/maintaining a fork, ask the upstream team for a release, or worst case, target a commit hash, rather than a version number in our dependencies.

**Comment by jerico** [comment]
Moving off Sprint 17.4 in favor of https://github.com/humanmade/product-dev/issues/1439

**Comment by johnbillion** [comment]
This was fixed in Stream 3.10, see #320

**Comment by mikelittle** [comment]
The new version of Stream has now been incorporated into altis-security.
